### PR TITLE
Changed 'host' and 'port' configuration to a new 'listen' style that …

### PR DESCRIPTION
**Pull request by mfrlin**
…is now supported by waitress server.

This should partly complete #2656

### CODE REVIEW

**Comment by stevepiercy** [comment]
@mfrlin please sign [CONTRIBUTORS.txt](https://github.com/Pylons/pyramid/blob/master/CONTRIBUTORS.txt) Remember to pull from Pylons/pyramid first to get the latest version of CONTRIBUTORS.txt. Otherwise LGTM.

**Comment by stevepiercy** [comment]
@mfrlin would you add a change note, too? https://github.com/Pylons/pyramid/blob/master/CHANGES.txt

**Comment by stevepiercy** [comment]
Thank you!

File: docs/tutorials/wiki2/src/basiclayout/production.ini [hunk 1]
```diff
@@ -16,8 +16,7 @@ sqlalchemy.url = sqlite:///%(here)s/tutorial.sqlite
 
 [server:main]
 use = egg:waitress#main
-host = 0.0.0.0
-port = 6543
+list = 0.0.0.0:6543
```
Comment (bertjwregeer) [inline docs/tutorials/wiki2/src/basiclayout/production.ini]: so much list

**Comment by bertjwregeer** [comment]
This should not have been merged without more carefully checking the PR for typo's and errors...

**Comment by bertjwregeer** [comment]
There are a bunch of `list` instead of `listen` in config files.

**Comment by bertjwregeer** [comment]
I've reverted this commit, follow the instructions in https://github.com/Pylons/pyramid/pull/2851 for updating this PR to be correct. Thanks.

**Comment by mfrlin** [comment]
There is a new PR here #2853

**Pull request merged**